.ci/jenkins/dockerfiles/ubuntu_16_04.dockerfile

```diff
@@ -60,4 +64,4 @@ RUN apt-get -y --no-install-recommends install \
     rm -rf /var/lib/apt/lists/*
 
 # Install tox
-RUN pip3 install tox
+RUN pip3 install tox
```

Comment:
Please add a new line to the end of the file.

Reply:
it has been already done :D